### PR TITLE
[cnsmap branch] Add CNSMAP content

### PR DESCRIPTION
**Pull request by lumjjb**

### CODE REVIEW

File: content/microservices-and-eliminating-implicit-trust.md [hunk 1]
```diff
@@ -7,9 +7,8 @@ category: "Runtime environment"
 The perimeter for containerized applications deployed as microservices is the microservice itself. Therefore, it is necessary to define policies that restrict communication only between sanctioned microservice pairs. The inclusion of zero trust in the microservice architecture reduces the blast radius by preventing lateral movement should a microservice be compromised. Operators should ensure that they are using capabilities such as network policies to ensure that east-west network communication within the container deployment is limited to only that which is authorized for access. There is some initial work done to provide strategies for microservices security through NIST SP 800-204 and may serve as a guide for implementing secure microservice architectures.
```
Comment (TheFoxAtWork) [inline content/microservices-and-eliminating-implicit-trust.md:7]:
```suggestion
The perimeter for containerized applications deployed as microservices is the microservice itself. Therefore, it is necessary to define policies that restrict communication only between sanctioned microservice pairs. The inclusion of zero trust in the microservice architecture reduces the blast radius by preventing lateral movement should a microservice be compromised. Operators should ensure that they are using capabilities such as network policies to ensure that east-west network communication within the container deployment is limited to only that which is authorized for access. There is some initial work done to provide strategies for microservices security through [NIST SP 800-204](https://doi.org/10.6028/NIST.SP.800-204) and may serve as a guide for implementing secure microservice architectures.
```

Comment (TheFoxAtWork) [reply content/microservices-and-eliminating-implicit-trust.md:7]: also - should this bulk text be 80 char break?  or will this effect the rendering in map?

Comment (lumjjb) [reply content/microservices-and-eliminating-implicit-trust.md:7]: shouldn't be, I think at the end of it we will run a linter on the entire thing and do it all in one PR

File: content/runtime.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ category: "runtime"
...
+Within the cloud native runtime environment, the microservice architecture for applications has been adopted by industries and organizations worldwide. Applications are often composed of several independent and single purpose microservices which communicate with each other via service layer abstractions which the container orchestration layer makes possible. Best practices to secure this interrelated component architecture involves ensuring that only sanctioned processes operate within a container namespace, prevention and notification of unauthorized resource access, and network traffic monitoring to detect hostile tooling activity. Service Mesh is another common abstraction that provides consolidated and complementary functionality for orchestrated services without imposing changes on the workload software itself (e.g. logging of API traffic, transport encryption, observability tagging, authentication, and authorization).
```
Comment (TheFoxAtWork) [inline content/runtime.md:9]: Should service mesh be capitalized here?

**Review by TheFoxAtWork — APPROVED** [review]
LGTM  !

File: content/storage-encryption.md [hunk 3]
```diff
@@ -25,14 +25,23 @@ and may be deployed with specific modes of encryption such as XTS for block devi
...
 -->
 
 ## Examples
-- TODO
-
-## Links
-- TODO
+- Ensure that storage services force the use of TLS/mTLS
+- Enable block storage encryption on the disk level (i.e. through dm-crypt)
+- Enable encryption encryption options of storage of filesystems and block storage solutions used, and when available, integrate the use of key management systems to manage keys
```
Comment (ashutosh-narkar) [inline content/storage-encryption.md:47]: Nit: `encryption` twice

**Review by ashutosh-narkar — APPROVED** [review]
LGTM

File: content/storage-encryption.md (hunk 3 above)
Comment (lumjjb) [inline content/storage-encryption.md:47]:
```suggestion
- Enable encryption options of storage of filesystems and block storage solutions used, and when available, integrate the use of key management systems to manage keys
```

**Pull request merged**